beacon-chain/rpc/eth/v1/validator/server.go

```diff
@@ -13,4 +14,5 @@ type Server struct {
 	TimeFetcher      blockchain.TimeFetcher
 	SyncChecker      sync.Checker
 	AttestationsPool attestations.Pool
+	V1Alpha1Server   *v1alpha1validator.Server
```

Comment:
Initially I wanted to implement https://github.com/prysmaticlabs/prysm/pull/9237 in order to reuse code between `eth/v1` and `prysm/v1alpha1`. Unfortunately the PR got blocked because it would interfere with Altair hard fork code. Instead of duplicating the whole logic I decided to make the `prysm/v1alpha1` server a part of the `eth/v1`` server. This is only a temporary solution because #9237 will eventually be merged.